pkg/embeddedyamls/yamls.go

```diff
@@ -3015,5 +3015,132 @@ roleRef:
   kind: Role
   name: submariner-metrics-reader
   apiGroup: rbac.authorization.k8s.io
+`
```

Comment:
This file is generated - you need to add yaml files under _config/rbac_ and then run `make pkg/embeddedyamls/yamls.go`.